views_euid/privacy.hbs

```diff
@@ -6,137 +6,159 @@
+      <li>If deemed necessary under applicable data protection legislation, the standard contractual clauses for data transfers, as approved by the European Commission Implementing Decision (EU) 2021/914 of 4 June 2021 for the transfer of Personal Data to third countries.</li>
+    </ul>
+    <p>If there is any conflict between the terms in this notice and such mechanisms, the terms of the transfer mechanism shall govern.</p>
+    <p>In compliance with the EU-U.S. DPF and the UK Extension to the EU-U.S. DPF and the Swiss-U.S. DPF, The Trade Desk commits to address unresolved complaints concerning our handling of personal data received in reliance on the EU-U.S. DPF and the UK Extension to the EU-U.S. DPF and the Swiss-U.S. DPF submitted to TRUSTe, an alternative dispute resolution provider based in the United States. If you do not receive timely acknowledgment of your DPF Principles-related complaint from us, or if we have not addressed your DPF Principles-related complaint to your satisfaction, please visit <a href="https://feedback-form.trustarc.com/watchdog/request">here</a> for more information or to file a complaint. The services of TRUSTe are provided at no cost to you.</p>
+    
     <h2>Data retention</h2>
-    <p>We store each EUID for up to 30 days from the date on which the EUID was created. If you withdraw your consent, or object to our processing your data, we keep a record of your objection for 5 years in order to implement your request.</p>
-    <p>We keep a hashed copy of the email addresses and phone numbers used to create an EUID. This is necessary for us to ensure compliance with your data protection rights.</p>
+    <p>We may store your hashed email address or phone number for up to 18 months from the date on which the EUID was created in order to operate the EUID for certain advertisers and/or data partners. If you withdraw your consent, or object to our processing of your data for EUID, we keep a record of your opt-out, a hashed copy of the email address or phone number used to create the EUID, and the EUID itself. This is necessary for us to implement your request and ensure ongoing compliance with your data protection rights.</p>
+    
     <h2>Contact us</h2>
     <p>You may contact us regarding privacy matters as follows:</p>
-    <p>Email:<b>  privacy  </b>((at))<b>  thetradedesk  </b>((dot))<b>  com</b></p>
-    <p><strong>Global Data Protection Officer (DPO)</strong> The Trade Desk, Global Privacy Office 42 N. Chestnut St., Ventura, CA 93001 Email: <b>dpo</b> ((at)) <b>thetradedesk</b> ((dot)) <b>com</b></p>
+    <p>Email:<b>  privacy  </b>((at))<b>  thetradedesk  </b>((dot))<b>  com</b> or submit an inquiry <a href="https://privacyportal-eu.onetrust.com/webform/2abc1a63-35c5-4ef7-b11b-1c55714738b5/4c5ba598-b05c-4b1c-b3da-abab6ac1c9db">here</a></p>
```

Comment:
Is there a reason this isn't just written as privacy@thetradedesk.com ? And/or linked?

Reply:
Just doing what I'm told.  This probably keeps the spam away from web scrapes looking for emails.

![image](https://github.com/user-attachments/assets/9d7d6317-2603-4cc1-9cf4-767912b67fcb)